Create a Fal Config File for fewer magic strings

Lightheart54 opened the pull request:
Removed the rest of the compiled files and consolidated Fal.Ai configuration into a json configuration file.

The API secret is now compiled into the project as a const property on the FalClientSettings Class.

DEV-RIOS commented:
You're absolutely correct on removing compiled files, however I do not like the design choice of loading the endpoints on runtime. This requires a 2nd file to run the plugin and while it seems small, I know will cause issues to users. 

Since we're already ingesting the api key during build from a config file, it makes sense to do so with endpoints. Will look into combining that all into the prebuild step.

Lightheart54 commented:
I completely understand. I was of both opinions on whether the api endpoints should be in source control or not. I've updated the PR to include a small Roslyn source code generator to generate a compiled class from the configuration json, removed the actual configuration file from source control, and updated the readme accordingly.
